ads/_config.js

```diff
@@ -98,6 +98,11 @@ export const adConfig = {
     prefetch: 'https://s.d.adup-tech.com/jsapi',
   },
 
+  affiliateb: {
+    prefetch: 'https://track.affiliate-b.com/amp/a.js',
+    renderStartImplemented: true,
+  },
+  
```

Comment:
to pass linter check, please remove trailing space.
you can run `gulp lint` locally to test